src/context.ts

```diff
@@ -77,60 +77,17 @@ export class Context {
     return this.tg
   }
 
-  get message() {
-    if (!('message' in this.update)) return undefined
-    return this.update.message
-  }
-
-  get editedMessage() {
-    if (!('edited_message' in this.update)) return undefined
-    return this.update.edited_message
-  }
-
-  get inlineQuery() {
-    if (!('inline_query' in this.update)) return undefined
-    return this.update.inline_query
-  }
-
-  get shippingQuery() {
-    if (!('shipping_query' in this.update)) return undefined
-    return this.update.shipping_query
-  }
-
-  get preCheckoutQuery() {
-    if (!('pre_checkout_query' in this.update)) return undefined
-    return this.update.pre_checkout_query
-  }
-
-  get chosenInlineResult() {
-    if (!('chosen_inline_result' in this.update)) return undefined
-    return this.update.chosen_inline_result
-  }
-
-  get channelPost() {
-    if (!('channel_post' in this.update)) return undefined
-    return this.update.channel_post
-  }
-
-  get editedChannelPost() {
-    if (!('edited_channel_post' in this.update)) return undefined
-    return this.update.edited_channel_post
-  }
-
-  get callbackQuery() {
-    if (!('callback_query' in this.update)) return undefined
-    return this.update.callback_query
-  }
-
-  get poll() {
-    if (!('poll' in this.update)) return undefined
-    return this.update.poll
-  }
-
-  get pollAnswer() {
-    if (!('poll_answer' in this.update)) return undefined
-    return this.update.poll_answer
-  }
```

Comment:
Redundant expressions to check if property exists in the object:
`if (!('something' in this.update)) return undefined`
We can use:
`return this.update?.something`
Which will return undefined if there is no `something`

Reply:
Optional chaining doesn't help, these are not null checks, they're narrowing an union type, hence CI fails.

Reply:
Optional chaining doesn't help, these are not null checks, they're narrowing an union type, hence CI fails.